src/app/home-page/widget/featured-content/featured-news.component.ts

```diff
@@ -0,0 +1,33 @@
+import { HttpClient } from '@angular/common/http';
+import { Component, OnInit } from '@angular/core';
+import { AlertService } from '../../../shared/alert/state/alert.service';
+import { Dockstore } from '../../../shared/dockstore.model';
+
+@Component({
+  selector: 'featured-news',
+  template: ` <div [innerHTML]="myExternalHTML"></div> `,
+})
+export class FeaturedNewsComponent implements OnInit {
+  // TODO should parameterize FeaturedContentComponent or something
+  public myExternalHTML: any = '';
```

Comment:
probably string type

Reply:
Amusing, switched the `any` to `string` and then it auto-formatted it out anyway

Reply:
interesting...i would've assumed the default would be any and then that would result in an `implicit any` lint issue (which we didn't enable because there would be a bajillion errors)